src/Components/SearchBar.js

```diff
@@ -45,7 +45,7 @@ if (Privacy.isAllowed(PRIVACY_ACTIONS.SEARCH)) {
         componentDidMount() {
             let options = {
                 query_by: this.props.query_by || 'name, runs, categories, web, slug, address, comments',
-                sort_by: 'sort-index:asc',
+                sort_by: '_text_match:desc,sort-index:asc',
```

Comment:
I can't find `_text_match` in the documentation. Did you mean `__match_score`?

Reply:
Interesting. The [guide](https://typesense.org/docs/0.14.0/guide/) talks about `_text_match` while [ŧhe API docs](https://typesense.org/docs/0.14.0/api/) talk about `__match_score`. Not sure what the difference is. `_text_match` seems to work fine and do what we want, though.

Reply:
Weird. The guide apparently mentions both, `_text_match` and `__match_score`. But it doesn't differentiate them. That worries me a bit, but ok. If it works, it should be fine I guess…

Reply:
Well, apparently using `_ match_score` produces an error, so lets just keep it that way.